README.md

```diff
@@ -1,6 +1,34 @@
-# Getting Started with Create React App
-
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# React Based Weather App
+This educational project is to bulid a mobile web app to check a list of metrics using React and Redux.
+An API that provides numeric data about a chosen topic is selected, and an app built around it. The app has 3 pages:
+- `Home` with a list of countries that can be filtered by region
+- `Cities` page with a list of cities for the selected country
+- `Details` page with the weather details. 
+The app gets the data from the [OpenWeather](https://openweathermap.org/api) api.
```

Comment:
Hi @felixodette , Kindly add a more descriptive README file like below and add a presentation video for review:

```
![](https://img.shields.io/badge/Microverse-blueviolet)

# Metrics App Capstone

> A crypto metrics app that pulls from the CoinGecko api to allow users view the various types of crypto currencies, view the current value and value history of crypto currencies.

![screenshot](./src/screenshot.png)

## Built With

- React & Redux
- Stylelint, Web Hint, Lighthouse
- Eslint, Prettier

## Video Demo

[Video Demo]()
## Live Demo

[Live Demo Link]()

## Getting Started
To get a local copy up and running, follow the steps in Setup below.

### Prerequisites
Basic knowledge of HTML, CSS, and JavaScript.

### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/felixodette/metrics-app/archive/refs/heads/main.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

```bash
git clone https://github.com/felixodette/metrics-app.git
```
```
- Once the files are on your machine, open the _todo-list_ folder in [Visual Studio Code](https://code.visualstudio.com/), and follow the steps in Installation & Set Up

## 🛠 Installation & Set Up

1. Install Node Modules

   ```sh
   npm install
   ```
```
2. Start the development server

   ```sh
   npm start
   ```
```
3. Open [http://localhost:3000](http://localhost:3000) to view it in the browser.

## 🚀 Building and Running for Production

1. Generate a full static production build

   ```sh
   npm run build
   ```
```
4. ## 🚀 Testing Application

1. Test the items Counter and Comments Counter function

   ```sh
   npm test
### Install

The following tools help make easier to work with sample code.

- [git](https://git-scm.com/downloads): A tool for managing source code
- [Visual Studio Code](https://code.visualstudio.com/): A source code editor
- [NVM](https://github.com/nvm-sh/nvm)

## **Temitope Ogunleye**

- GitHub: [@topeogunleye](https://github.com/topeogunleye)
- Twitter: [@tope_leye](https://twitter.com/tope_leye)
- LinkedIn: [ogunleye](https://linkedin.com/in/ogunleye)

## Acknowledgement

Original design idea by [Nelson Sakwa on Behance.](https://www.behance.net/gallery/31579789/Ballhead-App-%28Free-PSDs%29). Under the [Creative Commons License](https://creativecommons.org/licenses/by-nc/4.0/)
<br>

## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](https://github.com/felixodette/metrics-app/issues).

## Show your support

Give a ⭐️ if you like this project!


## 📝 License

This project is [MIT](./MIT.md) licensed.
```